src/rpcserver.cpp

```diff
@@ -292,8 +293,7 @@ static const CRPCCommand vRPCCommands[] =
     { "blockchain",         "verifytxoutproof",       &verifytxoutproof,       true  },
     { "blockchain",         "gettxoutsetinfo",        &gettxoutsetinfo,        true  },
     { "blockchain",         "verifychain",            &verifychain,            true  },
-    { "blockchain",         "invalidateblock",        &invalidateblock,        true  },
-    { "blockchain",         "reconsiderblock",        &reconsiderblock,        true  },
```

Comment:
We somehow missed https://github.com/bitcoin/bitcoin/commit/5ebe0956b391cfa782fd97bb417d20b479df6a36 so I resolved it this way.